recipes/raven-hydro/conda_build_config.yml

```diff
@@ -0,0 +1,4 @@
+libnetcdf:
+  - 4.6
+  - 4.7
+  - 4.8
```

Comment:
Do you need all those versions? If not you can remove this file and let conda-forge pinning build it for the latest one that is compatible with our stack.

Reply:
I think I can safely drop `4.6`, but this software is tightly coupled to `libnetcdf` (the version used to build the binary must also be available at runtime). `raven-hydro` is a base used in a fairly netcdf-heavy library, and I've run into issues where the most recent version supported is a few minor versions back. Would simply dropping 4.6 be alright?

Reply:
At the moment we build against 4.7, see https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L476, but there is an ongoing migration to 4.8. Don't worry about the coupling, it will always be build against the netcdf-c ABI that conda-forge pins. TL;DR that file is not needed.

Reply:
Done.